pallets/living-assets-ownership/src/traits.rs

```diff
@@ -41,15 +45,13 @@ pub trait CollectionManager<AccountId> {
 /// - `UnknownError`: An unspecified error occurred.
 #[derive(Debug, PartialEq)]
 pub enum CollectionManagerError {
-	CollectionAlreadyExists,
 	CollectionIdOverflow,
```

Comment:
Possible error not tested

Reply:
Also, we have two enums to manage errors. Do you think this can be improved somehow? One idea is to include a more generic enum (enum from trait) as an inner enum into the more specific one (enum from pallet).

Reply:
> Possible error not tested

removed

Reply:
I was considering using enums for nested errors, but that might be overengineering. For now, the way you've refactored it looks great